site.yml

```diff
@@ -53,7 +55,9 @@
     - role: inspec
       install_inspec_controls: [common]
       tags: [common,inspec]
-      when: inspec.enabled|default("False")|bool
+      when:
+        - inspec.enabled|default("False")|bool
+        - ursula_os == 'rhel'
```

Comment:
Yes, we should have separate PR to take care of this i.e
- remove password max days check from os_spec.rb 
- enable inspec common control for ubuntu and rhel.